client/analytics/components/graph/index.js

```diff
@@ -0,0 +1,124 @@
+/** @format */
```

Comment:
How about `/components/report-chart/index.js` with the component name `ReportChart` since it renders the Chart in the Report context?

A new folder for components we'd like to keep internal could be useful. Such as `/components-internal/report-chart/index.js`. Thoughts?